pkg/pillar/rootfs/init.sh

```diff
@@ -20,5 +20,4 @@ XENCONSOLED_ARGS='--log=all --log-dir=/var/log/xen' /etc/init.d/xencommons start
 sleep 5 # Let it come up
 
 echo 'Starting device-steps'
-/opt/zededa/bin/device-steps.sh >/var/log/device-steps.log 2>&1
-
+/opt/zededa/bin/device-steps.sh
```

Comment:
Hmm. Even device-steps.sh is logging to stdout? What happens in case rsyslogd dies during bootup?

Reply:
Nothing really -- messages just get delivered to memlogd in-memory ring buffer and are sitting there waiting for syslog to come back up and drain them.

Reply:
Why isn't LOG_TO_SYSLOG export removed from device-steps.sh? With this defined pillar services will keep sending logs directly to /dev/log

Reply:
It is removed in the overall commit that just went in